tests.js

```diff
@@ -54,6 +54,955 @@ window.Specs = {
+			"first-snapshot": 1998,
+			"last-snapshot": 1998
+		},
+		"properties": {
+			"letter-spacing": {
+				"links": {
+					"tr": "#propdef-letter-spacing",
+					"dev": "#propdef-letter-spacing"
+				},
+				"tests": ["normal", "10px"]
+			},
+			"text-align": {
+				"links": {
+					"tr": "#alignment-prop",
+					"dev": "#alignment-prop"
+				},
+				"tests": ["left", "right", "center", "justify"]
+			},
+			"text-decoration": {
+				"links": {
+					"tr": "#lining-striking-props",
+					"dev": "#lining-striking-props"
+				},
+				"tests": [
+					"none", "underline", "overline", "line-through", "blink",
+					"underline overline", "underline overline line-through"
+				]
+			},
+			"text-indent": {
+				"links": {
+					"tr": "#indentation-prop",
+					"dev": "#indentation-prop"
+				},
+				"tests": ["10px", "10%"]
+			},
+			"text-transform": {
+				"links": {
+					"tr": "#caps-prop",
+					"dev": "#caps-prop"
+				},
+				"tests": ["none", "capitalize", "uppercase", "lowercase"]
+			},
+			"white-space": {
+				"links": {
+					"tr": "#white-space-prop",
+					"dev": "#white-space-prop"
+				},
+				"tests": ["normal", "pre", "nowrap", "pre-wrap", "pre-line"]
+			},
+			"word-spacing": {
+				"links": {
+					"tr": "#propdef-word-spacing",
+					"dev": "#propdef-word-spacing"
+				},
+				"tests": ["normal", "10px"]
+			}
+		}
+	},
+
+	"css2-ui": {
+		"title": "CSS 2 User Interface",
+		"links": {
+			"tr": "CSS22/ui.html",
+			"dev": "css2/"
+		},
+		"status": {
+			"stability": "stable",
+			"first-snapshot": 1998,
+			"last-snapshot": 1998
+		},
+		"properties": {
+			"cursor": {
+				"links": {
+					"tr": "#cursor-props",
+					"dev": "#cursor-props"
+				},
+				"tests": [
+					"auto", "crosshair", "default", "pointer", "move", "e-resize",
+					"ne-resize", "nw-resize", "n-resize", "se-resize", "sw-resize",
+					"s-resize", "w-resize", "text", "wait", "help", "progress",
+					"url(cursor.png), auto", "url(cursor.svg), url(cursor.png), pointer"
+				]
+			},
+			"outline-color": {
+				"links": {
+					"tr": "#dynamic-outlines",
+					"dev": "#dynamic-outlines"
+				},
+				"tests": [
+					"black", "#00f", "#000000", "rgb(255, 255, 255)",
+					"rgb(100%, 50%, 50%)", "transparent"
+				]
+			},
+			"outline-style": {
+				"links": {
+					"tr": "#dynamic-outlines",
+					"dev": "#dynamic-outlines"
+				},
+				"tests": [
+					"none", "dotted", "dashed", "solid", "double", "groove",
+					"ridge", "inset", "outset"
+				]
+			},
+			"outline-width": {
+				"links": {
+					"tr": "#dynamic-outlines",
+					"dev": "#dynamic-outlines"
+				},
+				"tests": ["thin", "medium", "thick", "5px"]
+			},
+			"outline": {
+				"links": {
+					"tr": "#dynamic-outlines",
+					"dev": "#dynamic-outlines"
+				},
+				"tests": [
+					"black", "dotted", "5px", "#ff0000 dashed", "solid 0.2em",
+					"rgb(0, 0, 255) 0.1ex", "#0f0 double 0.8mm"
+				]
+			}
+		}
+	},
+
+	"css2-visudet": {
+		"title": "CSS 2 Visual Formatting Model",
```

Comment:
This section appears to be called "Visual Formatting Model Details"

Reply:
Corrected it.